kratos/sources/kratos_filesystem.cpp

```diff
@@ -68,15 +68,15 @@ void rename(const std::string& rPathFrom, const std::string& rPathTo)
 {
     return ghc::filesystem::rename(rPathFrom, rPathTo);
 }
-    
+
 std::string parent_path(const std::string& rPath)
 {
-    return ghc::filesystem::path(rPath).parent_path();
+    return ghc::filesystem::path(rPath).parent_path().string();
```

Comment:
this is necessary bcs in Win from what I understand the default is `w_string` hence we need the explicit conversion
see https://github.com/gulrak/filesystem/issues/94